src/main/java/uk/gov/hmcts/reform/civil/callback/CallbackHandler.java

```diff
@@ -102,4 +104,10 @@ protected CallbackResponse emptyCallbackResponse(CallbackParams callbackParams)
     protected CallbackResponse emptySubmittedCallbackResponse(CallbackParams callbackParams) {
         return SubmittedCallbackResponse.builder().build();
     }
+
+    public String getFullName(UserInfo userDetails) {
```

Comment:
not sure this is the right place for this method.

Reply:
both class extending this so why to duplicate or create another class

Reply:
fair. i was thinking more like json ignore in the data model place but not sure how GA project does stuff. Everything else LGTM.

Reply:
@gmikerlemos  move this method to util class